.env.test

```diff
@@ -5,9 +5,5 @@ FLASK_SKIP_DOTENV=True
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
-TRELLO_API_KEY=ABC
-TRELLO_API_TOKEN=DEF
-TRELLO_BOARD_ID=124
-TRELLO_LIST_TODO=567
-TRELLO_LIST_IN_PROGRESS=891
-TRELLO_LIST_DONE=345
\ No newline at end of file
+MONGO_CONNECTION_STRING=mongodb://fakemongo.com
+MONGO_DATABASE_NAME=testdb
```

Comment:
It would be good to also update your `.env.template` with similar

Reply:
I've added these env variables to the .env.template file.